src/ui/Carousel/index.tsx

```diff
@@ -0,0 +1,258 @@
+import './index.scss';
+import React, {ReactElement, useEffect, useRef, useState} from 'react';
+
+interface CarouselItemProps {
+  key: string;
+  item: ReactElement;
```

Comment:
Is it certain that the type of this is `ReactElement`?
Don't we need to define a separate type for this item?

Reply:
so the item is 
```
export const MessageTemplateWrapper = ({
  message,
  templateItems,
}: MessageTemplateWrapperProps): ReactElement => {
  return <MessageTemplateProvider message={message}>
    <MessageTemplate templateItems={templateItems} />
  </MessageTemplateProvider>;
};
```

but Im not sure if it is 
`(alias) const MessageTemplateProvider: (props: MessageContextProps) => React.JSX.Element`
or
`(alias) function MessageTemplate({ templateItems }: MessageTemplateProps): React.JSX.Element`